contributors/devel/bazel.md

```diff
@@ -20,22 +24,33 @@ tests, run
 $ bazel test //pkg/kubectl/...
 ```
 
+## Planter
+If you don't want to install Bazel, you can instead try using the unofficial
+[Planter](https://github.com/kubernetes/test-infra/tree/master/planter) tool,
+which runs Bazel inside a Docker container.
+
+For example, you can run
+```console
+$ ../test-infra/planter/planter.sh make bazel-test
+$ ../test-infra/planter/planter.sh bazel build //cmd/kubectl
+```
+
 ## Continuous Integration
 
-The [Bazel CI job](http://k8s-testgrid.appspot.com/google-unit#bazel) runs
-`make bazel-build`, `make bazel-test`, and (transitively) `make bazel-release`.
-A similar job is run on all PRs.
+There are several bazel CI jobs:
+* [ci-kubernetes-bazel-build](http://k8s-testgrid.appspot.com/google-unit#bazel-build): builds everything
+  with Bazel
+* [ci-kubernetes-bazel-test](http://k8s-testgrid.appspot.com/google-unit#bazel-test): runs unit tests in
+  with Bazel
 
-Many steps are cached, so the Bazel job usually executes fairly quickly.
+Similar jobs are run on all PRs; additionally, several of the e2e jobs use
+Bazel-built binaries when launching and testing Kubernetes clusters.
 
 ## Known issues
 
 [Cross-compilation is not currently supported](https://github.com/bazelbuild/rules_go/issues/70),
 so all binaries will be built for the host architecture running Bazel.
-Additionally, Go build tags are not supported. This means that builds on macOS may not work.
-
-[Binaries produced by Bazel are not statically linked](https://github.com/bazelbuild/rules_go/issues/161),
-and they are not currently tagged with version information.
+This means that builds on macOS may not work.
```

Comment:
Possibly mention planter as an unofficial work around for build-from-mac?
I'm also checking right now, do we know if builds currently work or not without planter?

Reply:
Confirmed that it fails on `pkg/proxy/ipvs/testing/BUILD` in a native macOS build as of `90a6a4f1f82cea1bbabde2f8b4e3dc786ff49ca2`

Reply:
done / elaborated

Reply:
oh, fails. ugh.